packages/babel-plugin-transform-es2015-arrow-functions/test/fixtures/arrow-functions/super-in-class-method/expected.js

```diff
@@ -0,0 +1,26 @@
+class SuperClass {
+  doStuff() {
+    return "superclass";
+  }
+}
+
+class MyClass extends SuperClass {
+  superDotAccess() {
+    var _super = super;
```

Comment:
This is unfortunately still a syntax error. `super` isn't a value you can assign to a variable. To my knowledge, our only option here would be

1. Throw if the arrow transform runs on and contains super inside a class
2. Have the arrow transform know how to implement `super` inside a class, e.g. `Reflect.get(SuperClass.prototype, 'doStuff', this).call(this);`.

Option 2 gets complicated real fast though, so I don't think I'm a fan of it. It also breaks down for `super()` still.